sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -827,7 +827,7 @@ def var_overload(df, axis=None, skipna=None, level=None, ddof=1, numeric_only=No
 
     Limitations
     -----------
-    Parameters ``axis``, ``level``, ``ddof`` and ``numeric_only`` unsupported.
+    Parameters ``axis``, ``level`` and ``numeric_only`` are supported only with default value ``None``.
```

Comment:
According to `check_type()` parameters `axis`, `level` and `numeric_only` don't always support default value `None`. So you need to change either the docstring or the code in typing stage.

Reply:
@densmirn Parameters axis, level, numeric_only only None or Ommited